assets/evaluation_on_cloud/environments/evaluations-built-in/context/Dockerfile

```diff
@@ -5,4 +5,5 @@ RUN pip install -r /app/requirements.txt
 
 # Copy your Python file into the image
 COPY evaluate_on_data.py /app/evaluate_on_data.py
-COPY save_evaluation.py /app/save_evaluation.py
\ No newline at end of file
+COPY save_evaluation.py /app/save_evaluation.py
+ADD online_eval /app/online_eval
```

Comment:
COPY?

Reply:
wondering why add instead of copy

Reply:
Add is needed to copy the entire directory